test/GnosisSafeMultisig.js

```diff
@@ -132,5 +132,63 @@ describe("GnosisSafeMultisig", function () {
             }
             expect(await multisig.getThreshold()).to.equal(threshold);
         });
+
+        it("Setting a guard contract address that guards transactions for not being able to transfer ETH funds", async function () {
+            // Set up a guard contract with the last Safe owner being the guard owner
+            const SafeGuard = await ethers.getContractFactory("SafeGuard");
+            const safeGuard = await SafeGuard.deploy(signers[2].address);
+            await safeGuard.deployed();
+
+            const to = safeGuard.address;
```

Comment:
Safe Guard address is the guard contract that is going to be called in the Safe modules setup, and is able to set the guard address slot value as it is called via a `delegatecall()` from the created Safe instance.